REDHOUSEVE/edx-platform
# Review Updated mapping for TinyMCE v5.4.2 in require-config.js

**Description:** 
Updated TinyMCE tag mapping in require-config.js

**JIRA:** 
https://edlyio.atlassian.net/browse/EDE-923

**Testing instructions:**

1. Open course page in studio
1. Click on `Schedule & Details` under `Settings` tab
1. Scroll Down and find the `Course Overview` section
1. The Course Overview section should be visible in TinyMCE
1. Change the content and save it
1. Visit the course about page on LMS, the change should be reflected there too
